instrumentation/aws-sdk/aws-sdk-2.2/library/src/test/java/io/opentelemetry/instrumentation/awssdk/v2_2/DirectLambdaTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v2_2;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import software.amazon.awssdk.services.lambda.model.InvokeRequest;
+
+public class DirectLambdaTest {
+  private Context context;
+
+  @Before
+  public void setup() {
+    GlobalOpenTelemetry.resetForTest();
+    ;
+    OpenTelemetrySdk.builder()
+        .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
+        .buildAndRegisterGlobal();
+
+    Span parent =
+        GlobalOpenTelemetry.getTracer("test")
+            .spanBuilder("parentSpan")
+            .setSpanKind(SpanKind.SERVER)
+            .startSpan();
+    context = parent.storeInContext(Context.current());
+    assertThat(context.toString().equals("{}")).isFalse();
```

Comment:
this is really roundabout. I would have asserted on `Span.fromContextOrNull(context)`